keyboard_rewrite/src/lib/y2_keyboard/clients/main.rb

```diff
@@ -0,0 +1,15 @@
+require_relative "../dialogs/layout_selector"
```

Comment:
I assume the usage of `require_relative` is just temporary until we have everything set up.

yast2-country is one of the oldest YaST modules. It still uses Makefile and it sucks regarding location of files, so `require_relative` is acceptable for the time being. But we should kill it at some point.

Reply:
Yeah it would be ideal to use require instead of require_relative, I will write it in a technical debt file, to remember it.